docstore/driver/util_test.go

```diff
@@ -79,7 +79,7 @@ func TestGroupActions(t *testing.T) {
 	}{
 		{
```

Comment:
Please add various cases involving atomic writes, mixed with other writes and gets before, after, and concurrently.